src/packages/@library/icons/ExternalLink.svelte

```diff
@@ -0,0 +1,18 @@
+<script>
+  export let height = "18px";
+  export let width = "18px";
+  export let color = "grey";
```

Comment:
make it to default 24px 24px grey, using this in all the icons to maintain consistency